python/pyspark/mllib/linalg.py

```diff
@@ -208,9 +209,55 @@ def __init__(self, ar):
             ar = ar.astype(np.float64)
         self.array = ar
 
+    def toString(self):
```

Comment:
This is not necessary.  In Python, we use __str__, and that's already implemented